dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -0,0 +1,349 @@
+
+The best way to read this page is side-by-side with [the AMP example in the repo](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp).
+
+<a name="amp-content-page" />
+
+### AMP content page
+
+The special meta tag in [amp_page.html](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/amp/amp_page.html) specifies the path to the publisher-hosted file to use as the AMP cross-domain host:
+
+```html
+<meta name="amp-3p-iframe-src" content="https://amp.publisher.com:5000/remote.html">
+```
+
+The `amp-ad` elements in the page body need to be set up as shown below, especially the following attributes:
+
++ *data-slot*: Identifies the ad slot for the auction.
++ *json*: Used to pass JSON configuration data across to the cross-domain host running the Prebid auction.  The format of the JSON is as shown: a `"prebid"` object containing the standard `"adUnits"` you know and love, plus a `"requestBidsDuration"` key which sets the Prebid auction timeout (in milliseconds).
+
+```html
+    <amp-ad
+      width="300"
+      height="250"
+      layout="fixed"
+      type="doubleclick"
+      json='{"prebid":{"requestBidsDuration":1000,"adUnits":[{"code":"/19968336/header-bid-tag-1","sizes":[[300,250],[300,600],[300,250],[100,100]],"bids":[{"bidder":"appnexusAst","params":{"placementId":"4799418","dealId":"some deal!"}},{"bidder":"aol","params":{"network":"10077.1","placement":3671670}},{"bidder":"sovrn","params":{"tagid":"315045"}}]},{"code":"/19968336/header-bid-tag-2","sizes":[[300,250],[300,600],[300,250],[100,100]],"bids":[{"bidder":"appnexusAst","params":{"placementId":"4799418","dealId":"some deal!"}},{"bidder":"aol","params":{"network":"10077.1","placement":3671670}},{"bidder":"sovrn","params":{"tagid":"315045"}}]}]}}'
+      data-slot="/19968336/header-bid-tag-1">
+    </amp-ad>
+```
+
+
+<a name="prebid.js-page" />
+
+### Prebid.js page
+
+The instructions below try to explain the code in [remote.html](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp/remote.html) in the [AMP integration example in our repo](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp).  When in doubt, use the AMP example files in the repo as your primary reference.
+
+The steps below are written to correspond roughly with the order in which you might write the code. For each step, we note whether the code should go in the page header or body.
+
+We recommend reading this side-by-side with [the AMP example files](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp) to see exactly how it's all laid out.
+
+This file is where the Prebid auction actually runs.
+
+#### 1. Add the AMP cross-domain iFrame source file (header)
+
+In your page's header, add the following JS in a script tag:
```

Comment:
In the AMP page header place a meta tag like this:
```
<meta name="amp-3p-iframe-src" content="https://amp.publisher.com:5000/remote.html">
```
See  
https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/amp/amp_page.html#L33